package.json

```diff
@@ -3,11 +3,13 @@
   "version": "0.25.0",
   "description": "programmatic plopping for fun and profit",
   "main": "lib/index.js",
-  "types": "index.d.ts",
+  "types": "types/index.d.ts",
   "scripts": {
     "clean": "node ./build-scripts/clean",
     "eslint": "eslint src/",
-    "test": "npm run eslint && npm run compile && ava",
+    "test": "npm run test:js && npm run test:ts",
+    "test:ts": "dtslint types --localTs node_modules/typescript/lib --expectOnly",
```

Comment:
should this be named as `test:types` perhaps just for clarity?

Reply:
Sure! I'm game to make that change :)